users/script.js

```diff
@@ -271,14 +271,19 @@ async function getParticularUserData(
 
 function showUserList(users) {
   const ulElement = document.createElement('ul');
-
+  if (!users.length) {
+    userListElement.innerHTML = 'No Users Found';
+    return;
+  }
```

Comment:
currently, this check will not work as our user API is also returning archived users. but a good thought to keep it.

Reply:
yeah this is. for the case when the applied filter doesn't gives any user.